src/ui/utils/autocomplete.ts

```diff
@@ -268,25 +223,48 @@ export function insertAutocompleteMatch(value: string, match: string, isArgument
     ? autocompleteLastArgument(value, autocompletedExpression)
     : autocompletedExpression;
 }
-export async function getAutocompleteMatches(input: string, scope: Scope) {
+export async function getAutocompleteMatches(
+  input: string,
+  scopes: ProtocolScope[],
+  fetchObjectWithPreview: ObjectFetcher
+) {
+  const currentScope = scopes[0];
+  if (!currentScope) {
+    return [];
+  }
   const propertyExpression = getPropertyExpression(input);
 
   if (propertyExpression) {
-    const properties = [];
+    const properties: string[] = [];
     const { left, right } = propertyExpression;
 
-    const value = getBinding(left, scope)?.value;
-    if (value) {
-      await value.traversePrototypeChainAsync(
-        current => current.loadIfNecessary(),
-        GETTERS_FROM_PROTOTYPES
+    // TODO This only handles top-level variables because of `=== left`.
+    // TODO Rework this to handle nested field lookups.
+    // TODO This also only looks in the _current_ scope.
```

Comment:
Might be nice for all of these TODO comments to have a Linear issue linked to them (so we don't just forget) ?

Reply:
Filed FE-761 with these TODO items